PollBuddy-Server/frontend/src/styles/main.scss

```diff
@@ -312,4 +312,76 @@ a:hover {
 
 .float-right {
   float: right;
-}
\ No newline at end of file
+}
+
+// Account Information Page
```

Comment:
Can you move these into an `AccountInfo.css` file instead? Also prefix the tags with `accountInfo-`.